tests/core/data/test_write_single_sheet_excel_data_node.py

```diff
@@ -51,6 +51,22 @@ def __post_init__(self):
                 setattr(self, field.name, field.type(value))
 
 
+def test_fail():
```

Comment:
This test implements the same logic as what we do in other tests, except it does not use the Taipy code.
It just simply reads an Excel file using `openpyxl.reader.excel.load_workbook`, closes it using `Workbook.close` and trees to delete the file.

The deletion raises an error.